src/main/scala/no/ndla/articleapi/controller/InternController.scala

```diff
@@ -86,12 +93,15 @@ trait InternController {
     }
 
     post("/id/article/allocate/?") {
+      authRole.assertHasRole(RoleWithWriteAccess)
```

Comment:
Dette betyr at alle som har skriverettigheter i draft-api også må ha skriverettigheter til article-api. Tror ikke det er et problem? Siden vi alikevel tenker å ha en rolle "articles:publish" som man må ha for å få lov til å publisere ting

Reply:
Godt poeng. Det er vel faktisk `drafts:write` det burde sjekkes på der. Synes ikke den skal stå åpen heller. InternController eller ikke føler jeg det er ryddigst å ha rollestyring på alt som er skriveoperasjoner.

Reply:
Enig 👍